src/targets/browser/browserPathResolver.ts

```diff
@@ -54,7 +55,15 @@ export class BrowserSourcePathResolver extends SourcePathResolverBase<IOptions>
 
     const unmappedPath = this.sourceMapOverrides.apply(url);
     if (unmappedPath !== url) {
-      return properResolve(webRoot, unmappedPath);
+      const webRootPath = properResolve(webRoot, unmappedPath);
+
+      // Prefixing ../ClientApp is a workaround for a bug in ASP.NET debugging in VisualStudio because the wwwroot is not properly configured
+      const clientAppPath = properResolve(webRoot, '..', 'ClientApp', unmappedPath);
+      if (!fs.existsSync(webRootPath) && fs.existsSync(clientAppPath)) {
```

Comment:
Can it be scoped to VS only, and possibly made async?

The original fix also had `leftPattern === 'webpack:///./*'` I dunno if you want to keep that and just do this inside the sourceMapPathOverrides code.

Reply:
Scoped to VS only. Made async. Added the webkpack prefix check.
sourceMapPathOverrides seems to be for all targets, I think it makes sense to leave this check on browserPathResolver which is browser specific (It doens't make sense to talk about webroot for the other targets).